addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -40,12 +40,22 @@ if (_checkVehicleIsStable) then {
 };
 
 private _radiusOfItem = 1;
-if (_typeOfCargo isKindOf "CAManBase") then {
+if (_cargo isKindOf "CAManBase") then {
     _radiusOfItem = 1.1;
 } else {
     //`sizeOf` is unreliable, and does not work with object types that don't exist on map, so estimate size based on cargo size
-    if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
-        _radiusOfItem = (((getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) ^ 0.35) max 0.75);
+    private _typeOfCargo = if (_cargo isEqualType "") then {_cargo} else {typeOf _cargo};
+    private _itemSize = if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
+        getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size));
+    } else {
+        if (["ace_cargo"] call EFUNC(common,isModLoaded)) then {
+            [_cargo] call EFUNC(cargo,getSizeItem);
+        } else {
+            _radiusOfItem;
+        };
+    };
+    if !(_itemSize isEqualTo -1) then {
+        _radiusOfItem = (_itemSize ^ 0.35) max 0.75;
```

Comment:
most significant change: before this PR if cargo module is not loaded `_radiusOfItem` was 1 and now it becomes `(1 ^ 0.35) max 0.75`.

Reply:
Ahah! I didn't know we could make joke in review!
Thanks a lot for the time you take to review this :-), I appreciate your feedback!

Reply:
`(1 ^ 0.35) max 0.75 = 1 max 0.75 = 1`

Reply:
Sorry I thought you were ironical :/